File: Darts/src/burlton/dartzee/code/screen/game/MatchStatisticsPanelGolf.java
package burlton.dartzee.code.screen.game;

import java.util.ArrayList;

public class MatchStatisticsPanelGolf extends GameStatisticsPanelGolf
{
	@Override
	protected void addRowsToTable()
	{
		super.addRowsToTable();
		
		addRow(new Object[getRowWidth()]);
		
		addRow(getBestGameRow(s -> s.min()));
		addRow(getAverageGameRow());
	}
	
	@Override
	protected ArrayList<Integer> getRankedRowsLowestWins()
	{
		ArrayList<Integer> rows = super.getRankedRowsLowestWins();
		rows.add(13);
		rows.add(14);
		return rows;
	}
}

File: src/main/kotlin/dartzee/screen/game/MatchStatisticsPanelGolf.kt
package dartzee.screen.game

import dartzee.core.util.minOrZero

class MatchStatisticsPanelGolf : GameStatisticsPanelGolf()
{
    override fun addRowsToTable()
    {
        super.addRowsToTable()

        addRow(getBestGameRow { s -> s.minOrZero() })
        addRow(getAverageGameRow())
    }

    override fun getRankedRowsLowestWins(): MutableList<String>
    {
        val rows = super.getRankedRowsLowestWins()
        rows.add("Best Game")
        rows.add("Avg Game")
        return rows
    }
}
